layouts/partials/job/gearing.html

```diff
@@ -21,15 +21,25 @@
       </div>
       <div class="xl:col-span-4 text-white">
         {{ with .Page.GetPage "stat-priority" }}
-          {{ $content := dict
-            "role" $role
-            "icon" "/theme-assets/materia.svg"
-            "name" "Stats & Materia"
-            "text" .Params.priority
-            "readMoreText" "Why this priority?"
-            "readMoreLink" "stat-priority"
-          }}
-          {{ partial "cards/default.html" $content }}
+          {{ if .Content }}
+            {{ $content := dict
+              "role" $role
+              "icon" "/theme-assets/materia.svg"
+              "name" "Stats & Materia"
+              "text" .Params.priority
+              "readMoreText" "Why this priority?"
+              "readMoreLink" "stat-priority"
+            }}
+            {{ partial "cards/default.html" $content }}
+          {{ else }}
+            {{ $content := dict
+              "role" $role
+              "icon" "/theme-assets/materia.svg"
+              "name" "Stats & Materia"
+              "text" .Params.priority
+            }}
+            {{ partial "cards/default.html" $content }}
+          {{ end }}
```

Comment:
I kinda dont like how im repeating code. I was trying to find out if Hugo has a way to set something like a "Global Variable" so I just have to update that in the IF statement...

Reply:
I think you should do a parent dict and then call "merge" on the extra options perhaps? Docs [here](https://gohugo.io/functions/merge/)

Reply:
I found another solution to use `.Scratch` instead to solve it. the `merge` has a problem of scoping when it is inside if statements unfortunately...

https://www.regisphilibert.com/blog/2017/04/hugo-scratch-explained-variable/